contributors/triage/README.md

```diff
@@ -0,0 +1,13 @@
+# Kubernetes Issue Triage
+
+This directory contains Kubernetes Issue Triage Guidelines and related documents.
+
+Triaging issues is a great way to get started contributing, and it will help you understand how the project is organized. Following are few predetermined searches for convenience:
+* [Longest Untriaged issues](https://github.com/kubernetes/kubernetes/issues?q=is%3Aissue+is%3Aopen+sort%3Acreated-asc) (sorted by age)
+* [Needs to be assigned to a SIG](https://github.com/kubernetes/kubernetes/issues?q=is%3Aissue+is%3Aopen+sort%3Acreated-asc)
+* [Newest incoming issues](https://github.com/kubernetes/kubernetes/issues?q=is%3Aopen+is%3Aissue)
+* [Busy Untriaged issues](https://github.com/kubernetes/kubernetes/issues?q=is%3Aissue+is%3Aopen+sort%3Acomments-desc) (sorted by number of comments)
+
```

Comment:
I'd like to have a `Least commented issues`. Few comments sometimes means less concern. They need more love.